README.md

```diff
@@ -277,7 +277,7 @@ Writing a new type definition is as simple as creating a new package. Start by c
 
 * **main** The entry point to the definition (canonical to "main" in NPM's `package.json`)
 * **browser** A string or map of paths to override when resolving (following the [browser field specification](https://github.com/defunctzombie/package-browser-field-spec))
-* **ambient** Denote that this definition _must_ be installed as ambient
+* **ambient** Denote that this definition should be installed as ambient to the most likely target environment
```

Comment:
I guess I'm just really confused reading it. It doesn't really flow or explain. 

> Denote that this definition should be installed as ambient to the most likely target environment

It should be installed as ambient (makes sense if you know what ambient is) but then "to the most likely target environment" is the part which isn't natural prose. It sounds like it's implying something that ambient does not do in this context.

Anyone else want to comment here? The phrasing isn't working for me, but I don't quite follow the gist of what's trying to be communicated (nothing personal). 

What's the issue with "Denotes the definition as an ambient dependency (E.g. a global dependency)". Something like that?